packaging/crosswalk-libs.spec

```diff
@@ -225,6 +225,10 @@ DEPENDENCIES=`python %{SOURCE1002} dump.json ${NINJA_TARGETS}`
 
 ninja %{?_smp_mflags} -C src/out/Release ${DEPENDENCIES}
 
+%if %{?BUILDDIR_NAME:1}%{!?BUILDDIR_NAME:0}
```

Comment:
Why do you think the check is needed here? Even if one's building in the default directory, removing this file should be harmless.